### ISSUE TITLE
When running against a few clusters with many pods with -o json I get "Error: couldn't parse this output"

### ISSUE THREAD

**Issue by robert-heinzmann-logmein** [jonnylangefeld/kubectl-mc#20]
When running kubectl mc against a few larger clusters with json output the command failes.

```
kubectl mc -r dev -o json -- get pods -A
```

I get the error: 
```
Error: couldn't parse this output. Are you sure your kubectl command allows for json output? Run command with -d to see debug output
```

When I run the same command without -o json, it works:

```
kubectl mc -r dev -- get pods -A
```

In this example the result set is 2331 pods distributed as follows: 

* Cluster1: 1619 pods
* Cluster2: 391 pods
* Cluster3: 317 pods

Could this be a buffer issue ?

**Comment by jonnylangefeld** [comment]
Hmm that's interesting! I'm wondering if somehow somewhere a (hidden) character makes its way into the output which lets the `json.Marsha` function fail.
Can you run your command with the `--debug` flag? Maybe the output of that helps, or you can post the output here.

**Comment by jonnylangefeld** [comment]
@robert-heinzmann-logmein could you give the latest version `v1.1.2` (via `kubectl krew upgrade mc`) another chance? I think I found the issue and fixed it in #21 
The reason was that all output to stderr was passed to stdout and therefore log lines like `Waited for 1.193922204s due to client-side throttling` were tried to be marshaled and that failed.

**Comment by robert-heinzmann-logmein** [comment]
Hi there @jonnylangefeld thank you for actually fixing this. I tested with 1.1.2 and I do not get this error anymore. Helped a lot, thanks.
